Security and Surveillance pull request

pradeepts opened the pull request:
Hi Team,

The pull request contains the following changes which are added to the core solution

1. Modularized templates for Basic, Standard and Premium 
2. Build code with Application Insights and OMS 
3. Templates with HA components
4. Added documentation(readme.md files ) for Updated solution

Please review the same and let us know if you need any changes.

Thanks,

githubmobiliya commented on documentation/Deployment-Guide.md:
```diff
@@ -0,0 +1,1136 @@
+| **OMS Location**   | Choose location for OMS Log Analytics to deploy. For more information,see https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-overview   | australiasoutheast, canadacentral, centralindia, eastus, japaneast, southeastasia, uksouth, westeurope  | eastus  |
+| **App Insights Location**  | specify the region for application insights   | eastus, northeurope, southcentralus, southeastasia, westeurope, westus2 | westus2 |
+| **Location DR**  | Choose locationDR which is different from the resource group region,if you have selected standard (or) premium option in the solution type inputparameter, for more information about the locationDr navigate to https://github.com/sysgain/SecurityAndSurveillance/blob/master/documentation/Deployment-Guide.md   | NA, EastUS2, CentralUS, WestUS2, WestCentralUS, CanadaCentral, Canad ...
+| **Traffic Manager Name**    | Please provide the name of the trafficmanager certficate custome domain name   |    | NA |
```
Typo "Custome" should be "custom"

pradeepts replied:
We have updated the above comment, please review and close it.

githubmobiliya commented on documentation/Deployment-Guide.md:
```diff
@@ -0,0 +1,1136 @@
+
+https://github.com/sysgain/SecurityAndSurveillance/tree/master
+
+2. Select **main-template.json** from **master** branch, click on **Raw** from the top right corner.
```
**main-template.json** --> not found in the repo.

pradeepts replied:
we have updated the correct file name i.e. maintemplate.json . Please review and close it.

githubmobiliya commented:
Please review the pull request for similar comments as above.

pradeepts commented:
We have reviewed the comments and updated accordingly, kindly review the same.

githubmobiliya commented on documentation/Deployment-Guide.md:
```diff
@@ -0,0 +1,1137 @@
+# Microsoft
+
+# Securiy Surveillance
```
Spell Correction Required: Security

pradeepts replied:
Updated the document with correct spelling.

githubmobiliya commented:
> Hi Team,
> 
> The pull request contains the following changes which are added to the core solution
> 
> 1. Modularized templates for Basic, Standard and Premium
> 2. Build code with Application Insights and OMS
> 3. Templates with HA components
> 4. Added documentation(readme.md files ) for Updated solution
> 
> Please review the same and let us know if you need any changes.
> 
> Thanks,

Hi Pradeep,

We tested deployment for basic plan and there were deployment errors.
Can you review and check the issues for the same. 

Attached maintemplate and parameters files for reference.

![Error1 (1)](https://user-images.githubusercontent.com/31727151/56113407-644e9b00-5f7b-11e9-822a-76eb50753ad8.png)

![Error1 (2)](https://user-images.githubusercontent.com/31727151/56113404-63b60480-5f7b-11e9-9baf-481cc150bca7.png)

![Error1 (3)](https://user-images.githubusercontent.com/31727151/56113442-86481d80-5f7b-11e9-8089-a87b4b197498.png)
[maintemplate.parameters.zip](https://github.com/MobiliyaTechnologies/SecurityAndSurveillance/files/3078810/maintemplate.parameters.zip)

pradeepts commented:
Hi Team,

As per the updates to the Azure, we have removed E2EDiagnostics, OMS for web application and added the following properties to the IoTHub resource.
{

                                "category": "DistributedTracing",

                                "enabled": true,

                                "retentionPolicy": {

                                    "days": 0,

                                    "enabled": false

                                }  

                            },

                            {

                                "category": "Configurations",

                                "enabled": true,

                                "retentionPolicy": {

                                    "days": 0,

                                    "enabled": false

                                }  

                            },

                            {

                                "category": "DeviceStreams",

                                "enabled": true,

                                "retentionPolicy": {

                                    "days": 0,

                                    "enabled": false

                                }  

                            }
After updating the above details we were able to deploy the template successfully. PFB Screenshot.
 
![MicrosoftTeams-image (24)](https://user-images.githubusercontent.com/20810624/56126355-9cb0a200-5f98-11e9-8487-25bcf6e9c7df.png)
![MicrosoftTeams-image (25)](https://user-images.githubusercontent.com/20810624/56126363-a20dec80-5f98-11e9-8025-5d69bf11d4ee.png)

The updated nested templates can be found in the same pull request. Please deploy the main-template.json and let us know if you need anything from us
_NOTE_: There is no change in the main-template.json file. The changes are made in the following nested templates(files) 
iothub.json
webapp.json
webappha.json
webapp-dr.json
